### ISSUE TITLE
Java support for automation api

### ISSUE THREAD

**Issue by everflux** [pulumi/pulumi-java#965]
## Hello!

- Vote on this issue by adding a 👍 reaction
- If you want to implement this feature, comment to let us know (we'll work with you on design, scheduling, etc.)

## Issue details
Provide Java support for the automation api so pulumi sdk can be used in Java based projects. 

### Affected area/feature

area/automation-api

**Comment by everflux** [comment]
Would it make sense to add this to https://github.com/pulumi/pulumi-java/issues/598 ?

**Comment by kennyjwilli** [comment]
Does supporting the automation api in Java necessitate https://github.com/pulumi/pulumi/issues/7219 being done?

**Comment by lukehoban** [comment]
> Does supporting the automation api in Java necessitate https://github.com/pulumi/pulumi/issues/7219 being done?

No. That should be orthogonal.  Automation API support for Java could be added today in a similar form as the Node.js, Python and .NET support.

> Would it make sense to add this to https://github.com/pulumi/pulumi-java/issues/598 ?

Yep - added!

**Comment by YanishR** [comment]
Hello 🙂 
Is there any update on the status of this issue? Will it be added to the roadmap soon? 
Thank you!

**Comment by justinvp** [comment]
Automation API is now available in the Java SDK 🎉. It shipped in version [v1.3.0](https://github.com/pulumi/pulumi-java/releases/tag/v1.3.0) of the SDK. More details: https://www.pulumi.com/blog/java-1-0/
